LoopKit/CarbKit/CarbMath.swift

```diff
@@ -425,6 +499,8 @@ fileprivate class CarbStatusBuilder<T: CarbEntry> {
     /// The carbohydrate-sensitivity factor for this entry, in glucose units per gram
     let carbohydrateSensitivityFactor: Double
 
+    /// The absorption time for this entry before any absorption is observed
+    let initialAbsorptionTime: TimeInterval
```

Comment:
`entry.absorptionTime` already contains this.

Reply:
`entry.absorptionTime` is optional, and `defaultAbsorptionTime` is not in `CarbStatusBuilder`, so I'd keep `initialAbsorptionTime` as is, no?